extensions/ql-vscode/src/vscode-tests/cli-integration/global.helper.ts

```diff
@@ -122,3 +125,34 @@ export async function cleanDatabases(databaseManager: DatabaseManager) {
     await commands.executeCommand("codeQLDatabases.removeDatabase", item);
   }
 }
+
+export async function fixWorkspaceReferences(
```

Comment:
Also wonder if you could add a comment here to describe why someone would use this

Reply:
Done